### ISSUE TITLE
WebRelay stopped working on 1.0.51

### ISSUE THREAD

**Issue by folosleg** [Ylianst/MeshCentral#4254]
**Describe the bug**
WebRelay was configured and working. After 1.0.51 update WebRelay server doesn't load upon MC startup.

**To Reproduce**
Auto-updated to 1.0.51

**Expected behavior**
Startup as with previous versions.

**Screenshots**
![image](https://user-images.githubusercontent.com/80481320/178546243-fcc96226-9e28-43fe-b5bb-1631c97c512f.png)

**Server Software (please complete the following information):**
 - OS: Ubuntu 20.04.04 LTS
 - Virtualization: bhyve
 - Network: WAN mode, HAProxy as reverseproxy & SSL offload
 - Version: 1.0.51
 - Node: 16.16.0
 - Browser: Google Chrome

**Your config.json file**
```JSON
{
  "$schema": "http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
  "settings": {
    "cert": "xx.xx.xx",
    "WANonly": true,
    "_LANonly": true,
    "sessionKey": "xxxx",
    "port": 444,
    "aliasPort": 443,
    "MpsPort": 0,
    "_MpsAliasPort": 4433,
    "_MpsAliasHost": "xx.xx.xx",
    "_MpsTlsOffload": true,
    "redirPort": 0,
    "_redirAliasPort": 80,
    "relayPort":454,
    "_relayAliasPort":443,
    "_relayDNS": "relay.xx.xx.xx",
    "tlsOffload": "xxx.xxx.xx.x",
    "desktopMultiplex": false,
    "webRTC": true,
    "webrtcConfig": {
      "iceServers": [
        { "urls": "stun:stun.nextcloud.com:443" },
        { "urls": "stun:stun.stunprotocol.org:3478" },
        { "urls": "stun:stun.3cx.com:3478" },
        { "urls": "stun:stun.l.google.com:19302" },
        { "urls": "stun:stun1.l.google.com:19302" }
      ]
    },
    "ClickOnce": true,
    "SelfUpdate": true,
    "CookieIpCheck": false,
    "_webPush": { "email": "xx@xx.xx" },
    "publicPushNotifications": true,
    "_InterUserMessaging": [ "user//xxx" ],
    "manageAllDeviceGroups": ["user//xxx"]
  },
  "domains": {
    "": {
      "title": "MeshCentral",
      "title2": "xx.xx",
      "minify": true,
      "hide": 4,
      "newAccounts": false,
      "_userNameIsEmail": true,
      "certUrl": "https://xx.xx.xx/",
      "_mstsc": true,
      "ssh": true,
      "altMessenging": { "name": "Jitsi", "url": "https://meet.xx.xx/xx-{0}" },
      "deviceMeshRouterLinks": {
        "rdp": true,
        "ssh": true,
        "scp": true,
        "extralinks": [
          {
            "name": "HTTP-Router",
            "protocol": "http",
            "port": 80,
            "_ip": "192.168.1.100",
            "_filter": [ "mesh/(domainid)/(meshid)", "node/(domainid)/(nodeid)" ]
          },
          {
            "name": "HTTPS-Router",
            "protocol": "https",
            "port": 443
          },
          {
            "name": "PSQL",
            "protocol": "psql",
            "port": 5432,
            "localport": 54321
          }
        ]
      }
    }
  },
  "_letsencrypt": {
    "__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before trying Let's Encrypt.",
    "email": "myemail@mydomain.com",
    "names": "myserver.mydomain.com",
    "production": false
  }
}
```

**Comment by si458** [comment]
you have `_relayDNS` set instead of `relayDNS`

**Comment by folosleg** [comment]
It was working with a port just fine, that's how I set it up in the first place. Ylian did not talk about the dns solution in his YT video.
I like the dns name idea better anyway. I started to set it up, but needed to config dns and reverseproxy. Did not have time, hence the commented out entry.
Will test the new release for the fix, and close this one here if all is good.

**Comment by Ylianst** [comment]
I broke `RelayPort` when I added `RelayDNS`, my fault, sorry about that. I am going to make a new release now...

**Comment by Ylianst** [comment]
MeshCentral v1.0.52 was just published, let me know if that solves it.

**Comment by folosleg** [comment]
Works like a charm! Thank you so much!

ps.: do you plan to implement one-to-many webRelays? is it possible at all? like relayDNS is *.relay.mc.com so each new relay gets a unique dns name generated so they can work in the same time within the same browser+login?

**Comment by Ylianst** [comment]
Nice and yes, multi-DNS web relay is already implemented in v1.0.52, you can specify an array of DNS names like this...

![MC2-WebRelayMultipleDnsNames](https://user-images.githubusercontent.com/1319013/178575786-c79d188d-3315-43c0-9d0e-d49cfe6bf71d.png)

**Comment by folosleg** [comment]
Absolutely amazing! Love your product! Cheers
